### PR TITLE
feat(cb2-10726): improve unit test coverage

### PR DESCRIPTION
**Pull request by JunYanBJSS**
## Tech Debit: Improve unit test coverage across VTM

Improve unit test coverage across VTM
[CB2-10726](https://dvsa.atlassian.net/browse/CB2-10726)

## Checklist

- [ ] Branch is rebased against the latest develop/common
- [ ] Code and UI has been tested manually after the additional changes
- [ ] PR title includes the JIRA ticket number
- [ ] Squashed commits contain the JIRA ticket number
- [ ] Delete branch after merge

### CODE REVIEW

**Review by naathanbrown — APPROVED** [review]

**Review by pbardy2000 — DISMISSED** [review]

File: src/app/app.component.spec.ts [hunk 1]
```diff
@@ -39,4 +45,86 @@ describe('AppComponent', () => {
     fixture.detectChanges();
     expect(app).toBeTruthy();
   });
+
+  describe('test route navigate', () => {
```
Comment (tomcrawleyy) [inline src/app/app.component.spec.ts:49]: following the pattern for other unit tests, the describe title should be the name of the function you are testing, so you should have individual describe blocks for router.initialNavigation & router.navigateByUrl

File: src/app/features/tech-record/components/adr-generate-certificate/adr-generate-certificate.component.spec.ts [hunk 2]
```diff
@@ -133,5 +133,30 @@ describe('AdrGenerateCertificateComponent', () => {
...
+
+      component.handleSubmit();
+      expect(dispatchSpy).toHaveBeenCalled();
+      expect(dispatchSpy).toHaveBeenLastCalledWith(
+        generateADRCertificate({ systemNumber: '123', createdTimestamp: '2021', certificateType: 'PASS' }),
+      );
+    });
+  });
+
+  describe('test get functions', () => {
```
Comment (tomcrawleyy) [inline src/app/features/tech-record/components/adr-generate-certificate/adr-generate-certificate.component.spec.ts:152]: describe title should be certificateTypes

File: src/app/app.component.spec.ts (hunk 1 above)
Comment (JunYanBJSS) [reply src/app/app.component.spec.ts:49]: Yup, thanks for the heads up. I am going to amend it to make it more sensible

File: src/app/forms/custom-sections/adr/adr.component.spec.ts [hunk 3]
```diff
@@ -41,4 +42,28 @@ describe('AdrComponent', () => {
       expect(spy).toHaveBeenCalled();
     });
   });
+
+  describe('handle form change', () => {
```
Comment (tomcrawleyy) [inline src/app/forms/custom-sections/adr/adr.component.spec.ts:46]: title should be camel case, ie handleFormChange to mimic the function name

**Review by SequoiaGod — APPROVED** [review]

**Review by pbardy2000 — APPROVED** [review]

**Pull request merged**